### ISSUE TITLE
Help! ClockingIt.com is down. Erlend's email also not working.

### ISSUE THREAD

**Issue by alexelov** [Fudge/clockingit#2]
Hello. ClockingIt.com is done. Instantly. Was working at Friday, 8th, 2022. The email I contacted normally in case any issues,
Erlend Simonsen <mr@fudgie.org> is not found as well. So sad. I have a feeling that may be the whole project is over. Im in panic as I not managed to do most latest backup only once per month. 
If anybody knows some more about all this please post a new issue with info.
I cant believe we lost a CIT, I been using it since 2010.
Alex

**Comment by davidreedernst** [comment]
I'm in exactly the same boat. I first noticed it being down on Sunday. The clockingit.com domain name is not returning an IP address, as if it expired (but it doesn't look expired).  fudgie.org is in the same state, so there won't be any emails working there. 

For routine maintenance, I usually try to contact @fudgie on Twitter.  This time, that's yielded nothing so far.  I'm honestly concerned that something might have happened to him.  

Anyway, Erlend, if you're reading this: my partner and I have a great deal of data stored in ClockingIT. I can imagine that you might not be interested in maintaining the site, which at this point is probably a nuisance that generates no revenue.  I can't blame you if so.  However, we would really really like to get access to our data there.  If the site could be brought back online even briefly to let me export some data, that would be great.  Or, honestly, just a dump of our data from the backend database would be immensely valuable to us.  We'd be willing to compensate you for your time if you're willing to work with us on this.  If nothing else, it would be very gratifying just to hear what's happened.  I would appreciate any response.
